Documentation/dev/libvirt-howto.md

```diff
@@ -81,7 +81,7 @@ echo server=/tt.testing/192.168.124.1 | sudo tee /etc/NetworkManager/dnsmasq.d/t
 ```sh
 go get github.com/crawford/terraform-provider-libvirt
 mkdir -p ~/.terraform.d/plugins
-cp $GOPATH/bin/terraform-provider-libvirt ~/.terraform.d/plugins/
+cp "$(go env GOPATH)/bin/terraform-provider-libvirt" ~/.terraform.d/plugins/
```

Comment:
Or it might be in `GOBIN`.  But unfortunately `go env GOBIN` does not give a sane default when the associated environment variable is unset.  And even without `GOBIN`, `GOPATH` could have multiple colon-delimited entries.  For something more(ish) reliable, see [the stuff starting here][1].  But that seems like a bit much for a HOWTO ;).  I'm fine leaving this as it stands, and expecting readers to adjust as needed.

Or maybe we can just set `GOBIN` for the one-off install?

```sh
GOBIN=~/.terraform.d/plugins go get github.com/crawford/terraform-provider-libvirt
```

That seems to work when I test it locally.

[1]: https://github.com/containers/libpod/pull/774/files#diff-b67911656ef5d18c4ae36cb6741b7965R39

Reply:
> I'm fine leaving this as it stands

You mean without this PR?



> Or maybe we can just set GOBIN for the one-off install?

`go env GOBIN` is empty on my system, so not a valid option.